samples/star-wars-api/Properties/launchSettings.json

```diff
@@ -1,27 +1,65 @@
 {
   "iisSettings": {
+    //#if (WindowsAuth)
+    "windowsAuthentication": true,
+    "anonymousAuthentication": false,
+    //#else
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
+    //#endif
     "iisExpress": {
-      "applicationUrl": "http://localhost:55802/",
+      "applicationUrl": "http://localhost:44299",
+      //#if (HasHttpsProfile)
+      "sslPort": 44300,
+      //#else
       "sslPort": 0
+      //#endif
     }
   },
   "profiles": {
-    "IIS Express": {
-      "commandName": "IISExpress",
+    //#if (HasHttpProfile)
+    "http": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
       "launchBrowser": true,
+      //#if (EnableOpenAPI)
+      "launchUrl": "swagger",
+      //#else
+      "launchUrl": "",
+      //#endif
+      "applicationUrl": "http://localhost:8086",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       }
     },
-    "FSharp.Data.GraphQL.Samples.StarWarsApi": {
+    //#endif
+    //#if (HasHttpsProfile)
+    "https": {
       "commandName": "Project",
+      "dotnetRunMessages": true,
       "launchBrowser": true,
+      //#if (EnableOpenAPI)
+      "launchUrl": "swagger",
+      //#else
+      "launchUrl": "",
+      //#endif
+      "applicationUrl": "https://localhost:8087;http://localhost:8086",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
-      },
-      "applicationUrl": "http://localhost:55803/"
+      }
+    },
+    //#endif
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      //#if (EnableOpenAPI)
+      "launchUrl": "swagger",
+      //#else
+      "launchUrl": "",
+      //#endif
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Does this file work in Visual Studio? Maybe we need to substitute the file on packaging?

Reply:
I don't get it. Explain in call what you mean